faq/templates/faq/inclusion_tags/faq_detail.html

```diff
@@ -2,12 +2,11 @@
 {% load ultracache_tags %}
 
 {% block content %}
-    {% ultracache 3600 "object-detail-inclusion" object.id object.modified request.page %}
+    {% ultracache 3600 "object-detail-inclusion" object.id object.modified %}
         {% with object.content_pages as content_pages %}
-            {% autopaginate content_pages 1 %}
             <div class="content">
                 <a name="jmbo-faq"></a>
-                {{ content_pages.0|safe }}
+                {{ content_pages|safe }}
```

Comment:
Mmm. I don't think FAQs should be multipage. That is functionality specific to posts. Just print `contebt|safe` and get rif of beautifulsoup and the content_pages property.